javaparser-core-testing/src/test/java/com/github/javaparser/JavaParserTest.java

```diff
@@ -32,29 +32,41 @@
 import com.github.javaparser.ast.type.ClassOrInterfaceType;
 import com.github.javaparser.ast.type.IntersectionType;
 import com.github.javaparser.ast.type.Type;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 
 import java.io.IOException;
 import java.nio.file.Path;
 import java.util.Optional;
 
 import static com.github.javaparser.ParseStart.COMPILATION_UNIT;
+import static com.github.javaparser.ParserConfiguration.LanguageLevel.*;
 import static com.github.javaparser.Providers.provider;
 import static com.github.javaparser.Range.range;
 import static com.github.javaparser.utils.CodeGenerationUtils.mavenModuleRoot;
 import static com.github.javaparser.utils.TestUtils.assertInstanceOf;
 import static com.github.javaparser.utils.Utils.EOL;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
 
 public class JavaParserTest {
 
+    @Before
+    public void setToLatestJava() {
```

Comment:
Actually I did the same in the pretty printer test in the PR you just merged. Can you please change it there too?
It thought there might be some way to create a custom JavaParser with custom configuration, so that you can have differently configured parsers (is there not?) and was too lazy to find out how as it was not instantly clear, so I thought I wait until you solve my comment here. :-D

Reply:
Also, if it is only needed for one of the tests in the class, it might be better to do it at start and end of the test method, not in `@Before` and `@After`, as that will do it for all methods in the test class.

Reply:
Oh, yes you can simply do new JavaParser() and get an instance (see http://javaparser.org/the-quick-and-the-full-api-of-javaparser/) but the static methods don't know about that instance :-/

Reply:
Looks like your PP tests are fine - am I missing something?